docs/source/augly.image.utils.rst

```diff
@@ -0,0 +1,37 @@
+augly.image.utils package
```

Comment:
same comment as audio lib -- i don't think we need to publicize this, as it's for internal development not external use (cc: @zpapakipos)